### ISSUE TITLE
Getting arrors when converting open template documents (Task did not complete within timeout)

### ISSUE THREAD

**Issue by SilverYoCha** [sbraconnier/jodconverter#186]
Hello,

I'm using your awesome API in order to convert kind of office documents from a Web Application (two LibeOffice instances are created at server start, on Windows 10).

It works for all documents i tried, except open template documents, like an **ott** for example (it works for **dotx** template).

I always get an exception **just** after this suite of instructions
`DocumentConverter.convert(source).to(destination).as(pdf).execute()`:

> Task did not complete within timeout: LocalConversionTask{source=SourceDocumentSpecsFromFile{file=writer.ott, format=ott}, loadProperties=null, target=TargetDocumentSpecsFromFile{file=file.pdf, format=pdf}, storeProperties=null}

I tried first with version 5.4.5.1 of LibreOffice and then with version 6.2.8.2, both without success and resulting with the exception I mentioned.

I don't really understand why as **ott** seems to be a supported format.
Did someone get this kind of issue and could tell me some stuffs to verify ?

Thank you.

**Comment by sbraconnier** [comment]
Is it possible for you to provide a sample file I could use to reproduce the problem ? Thanks!

**Comment by SilverYoCha** [comment]
Yes it is.
The zip archive contains one file (very simple) among which the problem always occurs in my environment.

[sample.zip](https://github.com/sbraconnier/jodconverter/files/4121378/sample.zip)

**Comment by sbraconnier** [comment]
I didn't have any problem. Here's my code (JodConverter 4.2.2):

```java
[writer.ott.pdf](https://github.com/sbraconnier/jodconverter/files/4136206/writer.ott.pdf)

import org.jodconverter.LocalConverter;
import org.jodconverter.office.LocalOfficeManager;
import org.jodconverter.office.OfficeUtils;

public class ConvertOtt {

  public static void main(String[] args) {

    LocalOfficeManager officeManager =
        LocalOfficeManager.builder()
            .install()
            .build();
    try {
      officeManager.start();

      File source = new File("C:\\temp\\sample\\writer.ott");
      File target = new File("C:\\temp\\sample\\writer.ott.pdf");
      LocalConverter.builder()
          .build()
          .convert(source)
          .to(target)
          .execute();
    } catch (Exception e) {
      OfficeUtils.stopQuietly(officeManager);
    }
  }
}
```

**Comment by SilverYoCha** [comment]
OK.

After several reboots of the machine these recent days, I tried again the conversion of the file that I gave as an example: it worked !!! (also with version 4.2.2 and LibreOffice 6.2.8.2)
I tried several template files and it generally worked well.
Sometimes I don't know why (I didn't investigate), I got an error ("Task failed").

So, I had surely missed something in my different tests before contacting you...
Thank you very much for your help and for your time.
